components/tooltip/test/tooltip.test.js

```diff
@@ -194,29 +194,22 @@ describe('d2l-tooltip', () => {
 
 	describe('delay', () => {
 
-		beforeEach(async() => {
-			tooltip.setAttribute('delay', 100);
-			await tooltip.updateComplete;
-		});
-
 		it('should not show if hover is lost before tooltip delay finishes', async() => {
-
-			oneEvent(tooltipFixture, 'd2l-tooltip-show').then(() => {
-				expect.fail('tooltip should not have been shown');
```

Comment:
This wasn't working at all and was just failing silently. It required an actual fix to tooltip.js below.